src/main/java/core/basesyntax/util/HibernateUtil.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
```

Comment:
I think we can probably remove this from the pull request)

Reply:
that's okay